### PR TITLE
Add miRBase client

### PR DESCRIPTION
**Pull request by cthoyt**
This PR adds a client for miRBase, which is a nomenclature for micro-RNAs. It adds a function for downloading and structuring the resource as a TSV that has cross-references to ncbigene and other organism specific databases. The client has lookup for terms by miRBase name, identifier, or HGNC. This PR also updates the PyBEL processor to handle nodes named with miRBase.

A few questions about adding this client:

- ~Should I add the resources in a versioned way? The .gitignore seems to reject stuff in the resources folder~ I added it with 8211c4a because I couldn't pass the tests and it looks like everything else is already added
- Should I update the hgnc_client to have a `is_mirna()` function like the `is_kinase()` and related?
- What else is necessary to finish #255?

### CODE REVIEW

**Comment by cthoyt** [comment]
Looks like the failing tests are from code that has to do with PubMed client... Unrelated to this PR

File: indra/resources/update_resources.py [hunk 1]
```diff
@@ -662,11 +664,13 @@ def _process_mirbase_file(lines):
...
             if not element.startswith('DR'):
                 continue
-            db, identifier, name = [e.strip() for e in element[len('DR'):].lstrip().split(';')]
+            db, identifier, name = [e.strip() for e in \
+                                       element[len('DR'):].lstrip().split(';')]
             yield mirbase_id, mirbase_name, db, identifier, name.rstrip('.')
 
 
 if __name__ == '__main__':
+    """
```
Comment (cthoyt) [inline indra/resources/update_resources.py]: Hey, what's going on here?

Comment (johnbachman) [reply indra/resources/update_resources.py]: D'oh, commented stuff out for testing and then forgot to uncomment 🤦‍♂

Comment (johnbachman) [reply indra/resources/update_resources.py]: Fixed

**Pull request merged**